toolchain/parse/typed_nodes.h

```diff
@@ -323,7 +328,8 @@ struct LetBindingPattern {
   static constexpr auto Kind = NodeKind::LetBindingPattern.Define(
       {.category = NodeCategory::Pattern, .child_count = 2});
 
-  NodeIdOneOf<IdentifierNameNotBeforeParams, SelfValueName> name;
+  NodeIdOneOf<IdentifierNameNotBeforeParams, SelfValueName, UnderscoreName>
```

Comment:
Would it be worth something like `using AnyBindingPatternName = NodeIdOneOf<IdentifierNameNotBeforeParams, SelfValueName, UnderscoreName>;` (noting there's a set of similar in node_ids.h)?